### DISCUSSION TITLE
[Communicate using Markdown] Course feedback survey!

### DISCUSSION THREAD

**Discussion by nguyenalex836** [skills/.github#548] [Communicate using Markdown]
Thank you for completing this course! ✨ We are delighted to have you as a user, and are committed to providing the best possible experience through GitHub Skills. 

Please take a moment to answer as many (or as few) questions as possible regarding the course you have just completed.

1. What part of the course did you enjoy the most?
2. What level of experience do you have with GitHub?
3. What’s something new you learned that you didn’t know before?
4. What’s something about the course that you didn’t enjoy as much?
5. What would you like to learn more about regarding GitHub?
6. What are you planning to use GitHub for?
7. How likely are you to recommend this course to a friend or colleague?

Your feedback is invaluable as we endeavor to enhance GitHub Skills for all GitHub users 💛

**Comment by Sithninja68** [comment]
What did I do…On Apr 4, 2024, at 2:24 PM, Alex Nguyen ***@***.***> wrote:﻿
Thank you for completing this course! ✨ We are delighted to have you as a user, and are committed to providing the best possible experience through GitHub Skills.
Please take a moment to answer as many (or as few) questions as possible regarding the course you have just completed.

What part of the course did you enjoy the most?
What level of experience do you have with GitHub?
What’s something new you learned that you didn’t know before?
What’s something about the course that you didn’t enjoy as much?
What would you like to learn more about regarding GitHub?
What are you planning to use GitHub for?
How likely are you to recommend this course to a friend or colleague?

Your feedback is invaluable as we endeavor to enhance GitHub Skills for all GitHub users 💛

**Comment by Developer354** [comment]
Github.com

**Reply by Developer354** [reply]
Is really good here

**Comment by kanhapise** [comment]
I am facing issue I did this skill but not able to mark as complete.

**Comment by RkBeau** [comment]
I also am having the same issue with this skill.  I have completed it (after a few false starts) but it will not accept the completion.

**Reply by sauravpaul2** [reply]
Any update on the same

**Comment by julioaranajr** [comment]
> 1. What part of the course did you enjoy the most?
- Adding images
> 2. What level of experience do you have with GitHub?
- Level = Continuous improvement -> Always curious
> 3. What’s something new you learned that you didn’t know before?
- Add a Code example and make a task list
> 4. What’s something about the course that you didn’t enjoy as much?
- Nothing, All good
> 5. What would you like to learn more about regarding GitHub?
- GitHub Sub-modules
> 6. What are you planning to use GitHub for?
- Personal, Community and Work
> 7. How likely are you to recommend this course to a friend or colleague?
- Very much

**Reply by nguyenalex836** [reply]
@julioaranajr Thank you very much for this feedback! ✨ 🚀

**Comment by jrandym** [comment]
> Thank you for completing this course! ✨ We are delighted to have you as a user, and are committed to providing the best possible experience through GitHub Skills.
> 
> Please take a moment to answer as many (or as few) questions as possible regarding the course you have just completed.
> 
> 1. What part of the course did you enjoy the most?
_seeing my Markdown file as a website!_
> 2. What level of experience do you have with GitHub?
_I've used it for personal projects for about 10 years and for teaching high school computer science for 6 years_
> 3. What’s something new you learned that you didn’t know before?
I learned how to publish my markdown files to a website
> 4. What’s something about the course that you didn’t enjoy as much?
_Nothing_
> 5. What would you like to learn more about regarding GitHub?
_I would like to learn how to contribute to open source projects_
> 6. What are you planning to use GitHub for?
_I plan to continue to use GitHub for personal projects and for teaching high school computer science_
> 7. How likely are you to recommend this course to a friend or colleague?
_Definitely!  I will probably give this course as a starter for my students_
> Your feedback is invaluable as we endeavor to enhance GitHub Skills for all GitHub users 💛
_You are welcome :-)_

**Comment by freddyfavour** [comment]
# Course Feedback Favour Alfred

## My review:
After going through this course I feel like I have been using markdowns in a very wrong way😅
I have been using GitHub for some while and couldn't really grasp the concept of markdowns, I even had to use utilize AI most times and still wasn't satisfied with the results.
Now,I'm confident that I can do anything with markdowns because of this course


## Course Feedback Survey

#### Q: What part of the course did you enjoy the most?
The part where I learnt how to add task and images was awesome and how the steps are well outlined.

#### Q: What level of experience do you have with GitHub?
I would say beginner,even though I've been using Github for about a year, I still don't understand most things and the things I understand are because of the courses from Github ; ) 

#### Q: What’s something new you learned that you didn’t know before?
How to add tasks and how the alt tag in images help screen-readers. 

#### Q: What’s something about the course that you didn’t enjoy as much?
Nothing,the course was packed with discovery and it was really nice.

#### Q: What would you like to learn more about regarding GitHub?
How to use Projects and Packages tabs

#### Q: What are you planning to use GitHub for?
For my projects in school as well as personal projects

#### Q: How likely are you to recommend this course to a friend or colleague?
A 100% probability♥

# Thanks a lot Github

**Comment by Mtettey29** [comment]
I was able to complete every task but unfortunately it was rejected

**Comment by cyberqubit** [comment]
How likely are you to recommend this course to a friend or colleague?
Yes of course nowadays GitHub is the very first step to start transitioning to coding and this is a core course that explains how to express beyond the code!

**Comment by tantowijh** [comment]
I cannot mark the course as complete, but I already finished it.

**Reply by unstoppableayush** [reply]
Same

**Reply by andonimarz** [reply]
Same problem

**Comment by unstoppableayush** [comment]
How to move forward after completing the markdown tasks?

**Reply by andonimarz** [reply]
I have the same problem but looks like there are not answers related

**Comment by xoraht** [comment]
1. What part of the course did you enjoy the most?
    Everything because I did these things first time.
2. What level of experience do you have with GitHub?
   Actually, I'm beginner.
3. What’s something new you learned that you didn’t know before?
   I learned how to use markdown what helped me to create my own README.md file.
4. What’s something about the course that you didn’t enjoy as much?
   Nothing.
5. What would you like to learn more about regarding GitHub?
   To be honest I don't know but i want to understand everything about how to use GitHub.
6. What are you planning to use GitHub for?
   Putting my project and learing new things.
7. How likely are you to recommend this course to a friend or colleague?
   I will recommend it to everybody who want to start using GitHub.

**Comment by Erick-Porto** [comment]
Honestly I liked the course as a whole, despite already knowing Github I didn't know how to do anything in it so everything I saw in the course was new, I learned practically everything. I would like courses explaining about project versioning too, it would be interesting. Currently I plan to use github as a place to search and store knowledge, make new connections and help as much as being helped.

**Comment by lilnurik** [comment]
I also have the same problem with this skill. After a few attempts, I completed it, but it still does not recognize the completion.

**Comment by JoseMiFlores** [comment]
I saw in the course was new, I learned practically everything.

**Comment by kdzamora** [comment]
What part of the course did you enjoy the most?
I didn't "enjoy" because things weren't explained clearly.
What level of experience do you have with GitHub?
Beginner, level 3 on a scale 1-5.
What’s something new you learned that you didn’t know before?
Some of the Markdown code features.
What’s something about the course that you didn’t enjoy as much?
I found the "Steps" confusing because I didn't realize that Github Actions were actually modifying the Readme file for each step. It finally clicked but that was very confusing for me. 
What would you like to learn more about regarding GitHub?
Why the authors chose to clutter the UI.  It is hard to focus and learn because there is so much going on.
What are you planning to use GitHub for?
Source code control.
How likely are you to recommend this course to a friend or colleague?
No, I believe a video explaining things would be much simpler.

**Comment by UntidyOctopus35** [comment]
I am so confused

**Comment by ktyler-git** [comment]
This was very hard to get through, as the directions are at best incomplete. 2 of 5 stars, as I spent too much time figuring out where I was supposed to be.

**Comment by Seifelden-Atia** [comment]
Thank you, it was a good introduction to Markdown! 
[Here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) are good docs for more info!

You need to add "Mark as completed" button to check completion of the course

![image](https://github.com/user-attachments/assets/4291f458-365c-4459-9d99-ecf4c59e9fbf)

**Reply by ktyler-git** [reply]
And where are those instructions??? I don't see that anywhere?

**Comment by Marcelo-Rub** [comment]
> Thank you for completing this course! ✨ We are delighted to have you as a user, and are committed to providing the best possible experience through GitHub Skills.
> 
> Please take a moment to answer as many (or as few) questions as possible regarding the course you have just completed.
> 
> 1. What part of the course did you enjoy the most?
> 2. What level of experience do you have with GitHub?
> 3. What’s something new you learned that you didn’t know before?
> 4. What’s something about the course that you didn’t enjoy as much?
> 5. What would you like to learn more about regarding GitHub?
> 6. What are you planning to use GitHub for?
> 7. How likely are you to recommend this course to a friend or colleague?
> 
> Your feedback is invaluable as we endeavor to enhance GitHub Skills for all GitHub users 💛

Es un buen curso para los que estamos aprendiendo, gracias

**Comment by QWeDev** [comment]
1. What part of the course did you enjoy the most?
Creating the markdown page.

2. What level of experience do you have with GitHub?
Beginner.

3. What’s something new you learned that you didn’t know before?
Deploying the markdown as a published page.

4. What’s something about the course that you didn’t enjoy as much?
Realizing I wasn't supposed to merge the pull request between tasks; started over and fixed it.

5. What would you like to learn more about regarding GitHub?
Ways that markdown can be made to look professional.

6. What are you planning to use GitHub for?
Professional web development as I learn and grow.

7. How likely are you to recommend this course to a friend or colleague?
Very likely.

**Comment by abdelmalek-create** [comment]
thank you

**Comment by Kubukhan** [comment]
What part of the course did you enjoy the most?
The whole process and learning new things.

What level of experience do you have with GitHub?
None prior to this but this course has given me beginner level experience.

What’s something new you learned that you didn’t know before?
I learnt how markdown can be used as a substitute
 
What’s something about the course that you didn’t enjoy as much?
I can't see any issues

What would you like to learn more about regarding GitHub?
how to create websites 

What are you planning to use GitHub for?
studying for cloud computing and it's a needed skill 

How likely are you to recommend this course to a friend or colleague?
very nice

**Comment by Caro-Sip** [comment]
> 1. What part of the course did you enjoy the most?
I enjoyed learning how to format code in md.

> 2. What level of experience do you have with GitHub?
Farely new.

> 3. What’s something new you learned that you didn’t know before?
Everything related to markdown.

> 4. What’s something about the course that you didn’t enjoy as much?
The process is too clunky. I understand this course made to be interactive for students, and new GitHub users but the implementation is not easy to use. As I was completing the course, I kept thinking these materials would be better if it was one website interactive than integrating it with Github.

> 7.     How likely are you to recommend this course to a friend or colleague?
Honestly, no, this course could be better but i was expecting something more but currently, the course doesn't really teach anything special that you wouldn't be able to find on a quick read through one of GitHub docs.

**Comment by kevinbrahyan** [comment]
Cool

**Comment by evoperator** [comment]
Cool course. Answering the questions:
1. IMAGES!
2. I already knew about markdown mainly because of using Discord
3. I didn't know <https://octodex.github.com> exists!
4. I got confused on step 4 because it said there's a new branch but it referred to the same one file, unless I read it wrong
5. Progressively more of everything until I tackle Git :)
6. Becoming a 10X dev of course
7. Friends, you need to check this one out just because

- [x] I enjoyed
- [x] I gave feedback
#### Good day
